src/form.jsx

```diff
@@ -278,7 +278,7 @@ export class Form extends React.Component {
   }
 
   componentDidMount = () => {
-    
+    this.getVisibleInputs()
```

Comment:
We need this method available to other components.

Two options:
1 - Create another action in redux that sets some variable to true. <Form/> will respond to it in componentWillReceiveProps and trigger this method.
2 - Pass in a copy of this method down below in the React children clone so that any children can call it.

#1 is ideal. #2 we can work with.